Google.Api.Generator.Rest/Models/PackageModel.cs

```diff
@@ -218,7 +216,11 @@ private ClassDeclarationSyntax GenerateServiceClass(SourceFileContext ctx)
                 var resourceProperties = Resources.Select(resource => resource.GenerateProperty(ctx)).ToArray();
 
                 var parameterizedCtor = Ctor(Modifier.Public, cls, BaseInitializer(initializerParam))(initializerParam)
-                    .WithBlockBody(resourceProperties.Zip(Resources).Select(pair => pair.First.Assign(New(ctx.Type(pair.Second.Typ))(This))).ToArray())
+                    .WithBlockBody(
+                        resourceProperties.Zip(Resources).Select(pair => pair.First.Assign(New(ctx.Type(pair.Second.Typ))(This)))
+                        .Append(baseUriProperty.Assign(Call("GetEffectiveUri")(IdentifierName("BaseUriOverride"), baseUriValue)))
```

Comment:
Nit: indent these three lines more? (So they're indented relative to `resourceProperties.Zip`?)

Reply:
Done